[ja] Add Japanese translation for getting started

ymotongpoo opened the pull request:
- This PR is for a documentation page whose authoritative copy is in the opentelemetry.io repository.
- 🛑 (@chalin) Blocked on #4627 

**Preview**: https://deploy-preview-4616--opentelemetry.netlify.app/ja/docs/getting-started/

chalin commented on content/ja/docs/getting-started/_index.md:
```diff
@@ -0,0 +1,30 @@
+---
+title: Getting Started
```
Should the title be translated too?

ymotongpoo replied:
Even in Japanese tech docs, it's common to have "getting started" as-is, but I put additional translation just in case.

chalin commented:
@ymotongpoo - hmm, there are issues with this PR. If you can, please to the following from your local environment:

- Ensure that youre `main` branch is at HEAD
- `pull` this PR (you might have to do a rebase on the pull)
- Rebase the branch of this PR from `main`
- Force push the branch of this PR

chalin reviewed (commented):
- I've submitted link-normalization updates to #4627 that will make it easier to keep the same link paths in this PR. So until that PR is merged, this one will be blocked (FYI)
- See inline comment.

ymotongpoo commented:
@chalin thank you for the comment. I force-pushed this branch. I hope it should fix the labels afterwards. (I don't have permission to change labels and GH Actions hasn't changed the labels yet.)

svrnm commented:
> @chalin thank you for the comment. I force-pushed this branch. I hope it should fix the labels afterwards. (I don't have permission to change labels and GH Actions hasn't changed the labels yet.)

I am not sure if the tooling is able to remove labels that have been set before, I manually removed them

ymotongpoo commented:
@chalin Hmm, I rebased my fork first, rebased it with upstream/main, force-pushed then I added a couple more commits. I can't figure out why you are not able merge this PR. Let me try another force push.

ymotongpoo commented:
@chalin I hope this push made it work.

chalin commented on content/ja/docs/getting-started/dev.md:
```diff
@@ -0,0 +1,36 @@
+
+次に、あなたが使っている[言語](/docs/languages/)のドキュメントを読んでください。
+
+- [C++](/docs/languages/cpp/)
+- [.NET](/docs/languages/net/)
+- [Erlang / Elixir](/docs/languages/erlang/)
+- [Go](/docs/languages/go/)
+- [Java](/docs/languages/java/)
+- [JavaScript / TypeScript](/docs/languages/js/)
+- [PHP](/docs/languages/php/)
+- [Python](/docs/languages/python/)
+- [Ruby](/docs/languages/ruby/)
+- [Rust](/docs/languages/rust/)
+- [Swift](/docs/languages/swift/)
+- [Other](/docs/languages/other/)
```
For this page and the others too, ensure that you apply the changes made via:

- #4627

ymotongpoo replied:
I see. I change them to point to ja pages.

chalin replied:
I'd rather that you use relative paths as is done in the `en` page.

ymotongpoo replied:
My understanding is that #4627 was merged 2 days ago, so if I rebase onto `main` yesterday, the change was applied. Is it correct?

Anyway, I misunderstood your comment so I reverted the path change.

chalin replied:
No worries @ymotongpoo. Merging of translation PRs is on hold. I'll get back to you as soon as I can.

svrnm commented:
@katzchang / @shirou, please review

katzchang reviewed (commented):
The Japanese is natural and looks good!

These documents are referenced from the top level `content/ja/_index.md`, will this be updated some other time?

svrnm commented:
> The Japanese is natural and looks good!

Thank you!
 
> These documents are referenced from the top level `content/ja/_index.md`, will this be updated some other time?

This should happen when the PR is merged (see the preview, if it does what you expect: https://deploy-preview-4616--opentelemetry.netlify.app/ja/)

@katzchang go ahead and create an issue for obtaining github org membership, via [this link](https://github.com/open-telemetry/community/issues/new?assignees=&labels=area%2Fgithub-membership&projects=&template=membership.md&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E), this should have been the case already when docs-ja was started, so use it as reference of your work + anything else (like this PR review) that you have contributed to the project.

katzchang commented:
I see I need to join the org membership to work as a reviewer, what should I do about 2 sponsors (one of two can be you @svrnm)?

svrnm commented:
> I see I need to join the org membership to work as a reviewer, what should I do about 2 sponsors (one of two can be you @svrnm)?

I sponsor you for sure, @ymotongpoo can sponsor you as well

ymotongpoo commented:
Thanks @svrnm for following up!

@katzchang Of course, I absolutely sponsor your membership for sure!

katzchang commented:
Awesome! I'll try.

ymotongpoo commented:
I deleted the remote branch by mistake. Reopening.

ymotongpoo commented:
@chalin @svrnm let me check the status. my understanding is that required steps to merge this and upcoming PRs are:

1. @katzchang gains the membership (https://github.com/open-telemetry/community/issues/2146)
2. `docs-ja-approvers` team will be created after 1.
3. @katzchang (or any other approvers if others gain the membership) signs off the PR
4. @chalin or @svrnm merges the PR

Is this correct?

svrnm commented:
> @chalin @svrnm let me check the status. my understanding is that required steps to merge this and upcoming PRs are:
> 
>     1. @katzchang gains the membership ([REQUEST: New membership for @katzchang community#2146](https://github.com/open-telemetry/community/issues/2146))
> 
>     2. `docs-ja-approvers` team will be created after 1.
> 
>     3. @katzchang (or any other approvers if others gain the membership) signs off the PR
> 
>     4. @chalin or @svrnm merges the PR
> 
> 
> Is this correct?

We are able to merge the PR without that being complete, we need to make this happen, but _technically_ the PR is ready, however all translation PRs will be on hold / throttled back (not only new ones), as @chalin works on resolving core issues with the i18n-support features.

chalin reviewed (changes requested):
@ymotongpoo - ok, I see what is happening. You've specified that this PR is based on the English files from commit 93ab1301. There have been updates to the pages in this PR since then.

Please use the following commit (or any more recent one) as your English-language file base: 

- e7a62891accc831de331336c8d10245969276112

You can see the diff here:

- https://github.com/open-telemetry/opentelemetry.io/commit/e7a62891accc831de331336c8d10245969276112#diff-2aa9d88e56f30c972fa5aeb63bcae20c8428572a6c6d2a229e57bafa72234628

chalin commented:
Also note that the Localization page instructions and script have changed. For the updated pages, see:

- https://opentelemetry.io/docs/contributing/localization/#tracking-changes-to-english-pages

By the time that you are done updating this PR, when you run `npm run check:i18n`, there should be no `ja` pages listed. Ok?

ymotongpoo commented:
Thank you @chalin for the confirmation. Because this is the PR for files under "Getting Started", I just updated the base revision of files under `getting-started` directory. 

Thanks to your script, I'm aware that the following files need to be updated similarly, which will be fixed in the following PRs.

```
> check:i18n
> scripts/check-i18n.sh

Processing paths: content
1       1       content/en/_index.md - content/ja/_index.md
30      30      content/en/docs/concepts/components.md - content/zh/docs/concepts/components.md
4       5       content/en/docs/_index.md - content/zh/docs/_index.md
1       1       content/en/_index.md - content/zh/_index.md
```

chalin commented:
> Thanks to your script, I'm aware that the following files need to be updated similarly, which will be fixed in the following PRs.

Yes, that is ok. For this PR I wanted to ensure that the Getting started pages reflected the latest `en` versions, which they do now, thanks for synchronizing these pages.

chalin commented:
Rebased PR that conforms to the new [Translation tips](https://opentelemetry.io/docs/contributing/localization/#translation-tips). All checks are passing, so I'll merge this now.

chalin reviewed (approved):
👍🏻
